app/indexers/file_set_indexer.rb

```diff
@@ -2,4 +2,54 @@
 
 class FileSetIndexer < Hyrax::FileSetIndexer
   self.thumbnail_path_service = ::ThumbnailPathService
+
+  def generate_solr_document
```

Comment:
was there previously no Solr document content generated for FileSets? Or are we overriding what was there and generating our own here?